### ISSUE TITLE
Lens Profile

### ISSUE THREAD

**Issue by franknitty69** [KonradIT/hero4hack#2]
is it possible for gpsend to issue a command to remove the stock lens profile?

**Comment by ghost** [comment]
This would help a lot of serious users who changed the stock lens to a non-fisheye one. It would make for a very serious video tool. Especially on drones. The current lens profile compensates for green corners on the stock lens, so when you change the lens you get images with slight pink corners. Removing or disabling the lens profile installed in the camera would eliminate this problem. 
There is a whole community waiting for a fix like this.

**Comment by KonradIT** [comment]
I am trying...

**Comment by franknitty69** [comment]
Thank you for all your work. Just knowing you are looking into this is huge.

**Comment by steveheinsch** [comment]
Yes, thank you very much for looking into it. There is a company that does this somehow, for a fee. They claim "Removing the lens profile is unaffected by future camera firmware updates" 
https://www.peauproductions.com/collections/services/products/service-remove-gopro-lens-profile-no-pink-corners

**Comment by PortlandPilot88** [comment]
Anyone ever make any progress on this?
